app.py

```diff
@@ -1,12 +1,36 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
-
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+#changed to the template for local host v2.0
+#unable to complete the sort by status function 
+
+#update from work 2.0
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', todos = items)
+    
+@app.route('/add-todo', methods=["POST"])
+def add_todo():
+    item = request.form.get('todo_task')
+    session.add_item(item)
+    return redirect('/')
+
+#delete function 
+@app.route('/delete-todo', methods=["POST"])
+def delete_todo():
+    item = request.form.get('todo_id')
```

Comment:
Variable name is slightly misleading - `item_id` or `todo_id` would be more accurate

Reply:
Probably my knowledge of this or lack of, but I am unsure here on what is misleading.  What have I named the variable here?

Reply:
You've called it `item`, and an item is something like
```
{ 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' }
```
whereas this is actually just the id. You've made the `item` vs `item_id` distinction elsewhere, e.g. the delete_item's parameter in its definition is `item_id`. So someone who isn't paying attention might think this variable called `item` is the full structure with all the properties.